html/themes/custom/common_design_subtheme/components/rw-homepage/rw-homepage.css

```diff
@@ -4,20 +4,27 @@
  * - tighter list
  * - hidden meta information
  */
+.rw-river--disasters {
+  --cd-flow-space: 1rem;
+  /* border-top: 1px solid #e6ecef; */
+}
 .rw-river--disasters .rw-river-article--disaster {
   position: relative;
   margin: 16px 0 0 0;
```

Comment:
This overrides the margin from the `cd-flow`.

Reply:
I'm not sure if you mean `cd-flow` from the base theme? It appears to be working the way I intended. Am I misunderstanding what you're pointing out?

Reply:
II mentioned that already in https://github.com/UN-OCHA/rwint9-site/pull/60#discussion_r689958035 and you changed it. It was just to point out that the `margin-top` from the `cd-flow` class didn't apply because of this `margin` rule.